frontend/src/components/editor-page/editor-pane/editor-pane.tsx

```diff
@@ -140,7 +140,7 @@ export const EditorPane: React.FC<ScrollProps> = ({ scrollState, onScroll, onMak
       <ToolBar />
       <ReactCodeMirror
         editable={firstUpdateHappened && connectionSynced}
-        placeholder={t('editor.placeholder') ?? ''}
+        placeholder={t('editor.placeholder', { host: window.location.host }) ?? ''}
```

Comment:
If there is a better way to do this in react, please let me know.

Reply:
This breaks in server side rendering because the window object doesn't exist there. Also this doesn't include port, protocol or subdirectory.
 
You should use the `useBaseUrl` hook to get the editor base url.

```typescript
const editorOrigin = useBaseUrl(ORIGIN.EDITOR)
```

Reply:
Thanks. Please review again